src/__tests__/integration/data/kv-storage.js

```diff
@@ -0,0 +1,75 @@
+export async function handle(state, action) {
+  const input = action.input;
+  const caller = action.caller;
+
+  if (!state.kvOps) {
+    state.kvOps = {};
+  }
+
+  if (!state.initialized) {
+    state.initialized = true;
+  }
+
+
+  if (input.function === 'mint') {
+    console.log('mint');
+    SmartWeave.kv.put(input.target, input.qty.toString());
+    // for debug or whatever
+    state.kvOps[SmartWeave.transaction.id] = SmartWeave.kv.ops();
```

Comment:
@jakub-wojciechowski not sure if we need, but it's possible to store all the operations on the KV storage that has been performed by an interaction.